tests/config/cluster_install/ols_manifests.yaml

```diff
@@ -187,3 +187,30 @@ rules:
     - "/ols-metrics-access"
   verbs:
     - "get"
+---
+# permissions to access the clusterversions object
```

Comment:
@bparees This is only on the test . But for the operator we already have `get` for `clusterversions`
```
			{
				APIGroups: []string{"config.openshift.io"},
				Resources: []string{"clusterversions"},
				Verbs:     []string{"get"},
			},
```
Do we need watch and list here @onmete ?

Reply:
Yes, I knew that someone already did these changes in operator at the time when we were solving data collector, as there was/is function accessing `clusterversions` resource to get the `cluster_id`. I think `get` should be enough, but I can't guarantee as I don't know much about that area.

Reply:
ok sounds good.  No, we do not need list+watch for clusterversion at present since the only thing we look at is the cluster_id and that should not change.